benchalerts/tests/unit_tests/expected_md/summary_nocommit.md

```diff
@@ -0,0 +1,10 @@
+Conbench analyzed the 2 benchmark runs that triggered this notification.
+
```

Comment:
Sorry for this comment being in the test and not at all attached to the code that generates this, but I think it's easier to discuss here in context:

I wonder if we should mention something about _why_ the commit hash isn't here. It "should be" relatively rare, but given the setup we are designing for, it'll come up semi-frequently. That might pre-empt questions about why conbench _sometimes_ links to a | describes the commit hash and _sometimes_ it doesn't

Reply:
Possibly we do this in a footnote or in the details somewhere, if we are worried about adding too much here.